Granicus.MediaManager.UserSDK/MediaManager.cs

```diff
@@ -61,6 +61,7 @@ public class MediaManager : SoapHttpClientProtocol, IMediaManager
         /// </remarks>
         public MediaManager()
         {
+            System.Net.ServicePointManager.SecurityProtocol = SecurityProtocolTypeExtensions.Tls12;
```

Comment:
Can we just do this:
                ServicePointManager.SecurityProtocol = (SecurityProtocolType)3072; // SecurityProtocolType.Tls12;
and avoid creating extensions?

Reply:
I gave that version a try and the tests pass on Win10 but fail on Win7. Think the extension method gives us a little more flexibility OS wise.
Also, seems to the MS preferred method:
https://support.microsoft.com/en-us/help/3154520/support-for-tls-system-default-versions-included-in-the-net-framework